spec/unit/service_spec.rb

```diff
@@ -368,44 +368,44 @@ module Backend
       describe '.active?' do
         it 'returns true when service is active' do
           [
-            { state: :active, id: '8001' },
-            { state: 'active', id: '8001' },
+            { state: :active, id: '8001', provider_key: 'foo' },
```

Comment:
I don't understand this change with adding `provider_key` to all `Service.save!` calls...

is this required now for some reason? I think it shouldn't?

Why were the tests passing without the provider key before?

Reply:
See this: https://github.com/3scale/apisonator/pull/384#issuecomment-2109914809
Then this: https://github.com/3scale/apisonator/pull/384#issuecomment-2110046219

Do you think the tests were right and it should be possible that `persist_sets` actually receives an empty provider hey?

Reply:
I've been checking, and it seems that indeed `provider_key` is an important part of the Service model on apisonator. I thought it wasn't the case, because `service_token` is used for most operations on the Service Management API, but it seems that when Service is created or updated through the internal API, the`provider_key` is indeed provided :grimacing: https://github.com/3scale/porta/blob/4b2eb62c3acea37ec32b0f1339a2d63f6b8ddef1/app/lib/backend/model_extensions/service.rb#L20

So, I guess it's fine to add it to tests.